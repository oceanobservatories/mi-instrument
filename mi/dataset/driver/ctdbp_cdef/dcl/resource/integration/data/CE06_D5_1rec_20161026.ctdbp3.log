2016/10/26 00:28:02.804 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/10/26 00:30:14.663 [ctdbp3:DLOGP6]:Instrument Started on second attempt [Power On]
2016/10/26 00:31:07.452 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2016/10/26 01:28:04.264 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/10/26 02:28:03.698 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/10/26 03:28:03.965 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/10/26 03:30:15.778 [ctdbp3:DLOGP6]:Instrument Started on second attempt [Power On]
2016/10/26 03:31:08.567 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2016/10/26 04:28:05.314 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/10/26 04:30:03.883 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2016/10/26 04:31:05.484 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2016/10/26 05:28:02.978 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/10/26 06:28:03.875 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/10/26 07:28:04.346 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/10/26 08:28:04.575 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/10/26 09:28:03.879 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/10/26 10:28:07.523 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/10/26 11:28:02.934 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/10/26 12:28:06.148 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/10/26 13:28:01.940 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/10/26 14:28:03.761 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/10/26 15:28:05.317 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/10/26 15:30:08.040 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2016/10/26 15:31:09.635 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2016/10/26 16:28:06.000 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/10/26 17:28:03.121 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/10/26 18:28:03.658 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/10/26 19:28:03.682 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/10/26 20:28:03.947 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/10/26 21:28:04.921 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/10/26 21:30:16.649 [ctdbp3:DLOGP6]:Instrument Started on second attempt [Power On]
2016/10/26 21:30:31.369  12.9060,  3.28307,    0.985, 990, 171, 91, 26 Oct 2016 21:30:25
2016/10/26 21:31:06.569 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2016/10/26 22:28:04.528 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/10/26 22:30:16.390 [ctdbp3:DLOGP6]:Instrument Started on second attempt [Power On]
2016/10/26 22:31:09.179 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2016/10/26 23:28:04.608 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
