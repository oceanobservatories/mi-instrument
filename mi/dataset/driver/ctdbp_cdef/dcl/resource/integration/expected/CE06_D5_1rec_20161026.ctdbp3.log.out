
 subsite  | node  | sensor       | bin        | method      | time                 | deployment | internal_timestamp | port_timestamp       | preferred_timestamp | quality_flag | conductivity | date_time_string | dcl_controller_timestamp | pressure | temp
----------+-------+--------------+------------+-------------+----------------------+------------+--------------------+----------------------+---------------------+--------------+--------------+------------------+--------------------------+----------+--------
 CE06ISSM | SBD17 | 06-CTDBPC000 | 3686428800 | telemetered | 3686506231.368999958 |          5 |         3686506225 | 3686506231.368999958 |      port_timestamp |           ok |      3.28307 |             null |                     null |    0.985 | 12.906

(1 rows)
